### PR TITLE
Fixed dice loss to use probabilities, not logits.

### PR DESCRIPTION
**Pull request by Zach-ER**
I have changed the Dice loss to use the probabilities instead of the logits.

### CODE REVIEW

**Comment by Zach-ER** [comment]
this implementation doesn't work yet...

**Comment by jakeret** [comment]
Sounds interessting. Looking forward for your PR.
J
